content/api/metrics.apib

```diff
@@ -49,7 +49,16 @@ Definitions for terms found in Metrics API
 | `total_delivery_time_first`      | Total time taken to deliver messages on first attempt (milliseconds)                                |
 | `total_delivery_time_subsequent` | Total time taken to delivery messages on subsequent attempts (milliseconds)                         |
 | `count_unsubscribe`              | Total number of unsubscribes as a result of clicked links and the ISP list unsubscribe feature      |
+| `count_inbox_panel`              | Panel messages delivered to the inbox.                                                              |
```

Comment:
Prefix the descriptions with the Deliverability tag (should link to the product page or wherever @haleysolomon thinks it should redirect to)